common/safecrt.c

```diff
@@ -66,7 +66,7 @@ oe_result_t oe_memset_s(void* dst, size_t dst_size, int value, size_t num_bytes)
     volatile unsigned char* p = dst;
 
     if (dst == NULL)
-        OE_RAISE(OE_INVALID_PARAMETER);
+        OE_RAISE_NO_TRACE(OE_INVALID_PARAMETER);
```

Comment:
It seems strange that general purpose CRT routines would decide to do no-trace

Reply:
Good comment!
I was trying to remove unnecessary error messages but this change might have over-done it.
I will put them back.